content/developers/metrics/distributions.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Distributions
+kind: documentation
+further_reading:
```

Comment:
We should add a link to https://docs.datadoghq.com/graphing/metrics/distributions/ on this page to encourage them to learn more about the dedicated UI for distribution metrics.